### ISSUE TITLE
lightgbm.basic.LightGBMError: Could not open

### ISSUE THREAD

**Issue by pasa13142** [serengil/deepface#47]
Hello Sefik!

Firstly thanks for this amazing project,

I was using your ensemble face verification system but today i can not use it because of getting this error:

lightgbm.basic.LightGBMError: Could not open /home/comp/anaconda3/envs/verification/lib/python3.6/site-packages/models/face-recognition-ensemble-model.txt

There is no model directory in site-packages. This occured today. Maybe smth changed in lightgbm. I looking for how to figure it out but cant find anything.

You have any idea ? 

Have a nice day

**Comment by serengil** [comment]
Hello, 

model.txt is in the repo. You can check it here: https://github.com/serengil/deepface/tree/master/models


Could you update /home/comp/anaconda3/envs/verification/lib/python3.6/site-packages/deepface/DeepFace.py 's 126th line as below?

`deepface_path = deepface_path.replace("\\", "/").replace("/DeepFace.py", "")`

**Comment by pasa13142** [comment]
Thanks for your quick response,

it is solved by both changed the line as you mentioned and paste model directory in your repo to under site-packages/deepface/. smth like site-packages/deepface/models.

it is about missing model file and i didnt realize it is in the repo. by adding it, it is solved.

Thanks a lot

**Comment by serengil** [comment]
This means that I need to update the repo as well :) Thank you.

**Comment by serengil** [comment]
this is fixed in 0.0.27

**Comment by pengKiina** [comment]
Hi Selfik,

by using the deepface 0.0.31, it also occurs the following error, please help take a look.

Thanks. 

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-42-fb9648fe7acb> in <module>
----> 1 resp_obj = DeepFace.verify(path + "yang1.jpeg", path +"yang2.jpg", model_name = "Ensemble")
      2 df2 = DeepFace.find(img_path = path + "img1.jpg", db_path = path, model_name = "Ensemble")

c:\users\pysep\appdata\local\programs\python\python38\lib\site-packages\deepface\DeepFace.py in verify(img1_path, img2_path, model_name, distance_metric, model, enforce_detection)
     39                 print("Ensemble learning enabled")
     40 
---> 41                 import lightgbm as lgb #lightgbm==2.3.1
     42 
     43                 if model == None:

ModuleNotFoundError: No module named 'lightgbm'
```
